tests/time_tests/scripts/run_timetest.py

```diff
@@ -27,35 +26,11 @@
 
 from test_runner.utils import filter_timetest_result
 
+UTILS_DIR = os.path.join(Path(__file__).parent.parent.parent, "utils")
```

Comment:
I would prefer to do timetests refactoring in separate PR in order to not to combine a lot of changes here